pkgs/tools/misc/nagstamon/default.nix

```
@@ -1,27 +1,47 @@
-{ lib, fetchurl, pythonPackages }:
+{ lib, fetchFromGitHub, python39Packages, wrapQtAppsHook }:
```

```suggestion
{ lib, fetchFromGitHub, python3Packages, wrapQtAppsHook }:
```

and then remove passing pythonPackages in `pkgs/top-level/all-packages.nix

```
  nagstamon = callPackage ../tools/misc/nagstamon {
    pythonPackages = python3Packages;
  };
```